config.example.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var config = {}
+config.session = {}
```

Comment:
You pointed to the file that contained the change 😕 It was breaking for me. IDK maybe a node version thing?